testDataSetup/mongo/mongo.go

```diff
@@ -114,34 +114,35 @@ func SetupMany(d *ManyDocs) error {
 	return nil
 }
 
-// FilterJob represents a structure for a filter job
-type FilterJob struct {
-	InstanceID       string      `bson:"instance_id"          json:"instance_id"`
-	DimensionListURL string      `bson:"dimension_list_url"   json:"dimension_list_url,omitempty"`
-	Dimensions       []Dimension `bson:"dimensions,omitempty" json:"dimensions,omitempty"`
-	Downloads        *Downloads  `bson:"downloads,omitempty" json:"downloads,omitempty"`
-	Events           *Events     `bson:"events,omitempty"    json:"events,omitempty"`
-	FilterID         string      `bson:"filter_job_id"        json:"filter_job_id,omitempty"`
-	State            string      `bson:"state"                json:"state,omitempty"`
-	Links            LinkMap     `bson:"links"                json:"links,omitempty"`
+// Filter represents a structure for a filter job
```

Comment:
surely, now `// Filter represents a structure for a filter blueprint or output`

Reply:
👍